init/setup-rw.sh

```diff
@@ -25,33 +25,31 @@ if mountpoint -q /rw ; then
 # This script will be executed at every VM startup, you can place your own
 # custom commands here. This include overriding some configuration in /etc,
 # starting services etc.
-#
-# You need to make this script executable to have it enabled.
 
 # Example for overriding the whole CUPS configuration:
 #  rm -rf /etc/cups
 #  ln -s /rw/config/cups /etc/cups
 #  systemctl --no-block restart cups
 EOF
+        chmod 755 /rw/config/rc.local
 
         touch /rw/config/qubes-firewall-user-script
         cat > /rw/config/qubes-firewall-user-script <<EOF
 #!/bin/sh
 
-# This script is called in ProxyVM after firewall every update (configuration
```

Comment:
"firewall" word was important here - this script isn't called on any update, just after firewall change.

Reply:
Ah, makes sense.

I force-pushed a new version re-adding the word "firewall"